packages/core/parcel/package.json

```diff
@@ -37,4 +37,4 @@
   "devDependencies": {
     "@parcel/babel-register": "2.0.0-beta.1"
   }
-}
+}
```

Comment:
...

Reply:
This is one of the unnecessary changes I was talking about. 

If either one of you is using Refined GitHub, you might be able to undo every file that has unrelated changes (however if the change is “add a trailing newline”, I think that it won’t be able to undo it)

Reply:
This is my fault, I never add newlines at the end of files and my code editor (VSCode) refuses to as well. Sorry for all the trivial mistakes.